### PR TITLE
Replace legacy vsphere docs with agent docs

### PR DESCRIPTION
**Pull request by pmcollins**

### CODE REVIEW

File: vsphere/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+browse_categories:
+- Cloud infrastructure and services
+data_signature: vcenter:"*"
```
Comment (asuresh4) [inline vsphere/meta.yaml]: IIRC we typically refrain from having wildcards here. @mstumpfx ?

**Comment by asuresh4** [comment]
@pmcollins could you also add the new `metrics.yaml` file?

**Comment by jaysfx** [comment]
> @pmcollins could you also add the new `metrics.yaml` file?

Looks like `scripts/docs/to-integrations-repo` should be creating it right?

**Comment by asuresh4** [comment]
> > @pmcollins could you also add the new `metrics.yaml` file?
> 
> Looks like `scripts/docs/to-integrations-repo` should be creating it right?

Yes, the script should generate it.

File: vsphere/meta.yaml (hunk 1 above)
Comment (pmcollins) [reply vsphere/meta.yaml]: Ok, fixed.

**Pull request merged**